test/fixtures/infra/aws-apigatewayv2/cfn_api.ts

```diff
@@ -0,0 +1,33 @@
+import { CfnApiProps } from "@aws-cdk/aws-apigatewayv2";
```

Comment:
Would it be worth adding a note to the resource_prefixer readme for our future selves about where to find these fixture examples, given the difficulties we had?